services/telemetry/telemetry.go

```diff
@@ -389,6 +389,7 @@ func (ts *TelemetryService) trackConfig() {
 		"enable_gif_picker":                                       *cfg.ServiceSettings.EnableGifPicker,
 		"gfycat_api_key":                                          isDefault(*cfg.ServiceSettings.GfycatAPIKey, model.ServiceSettingsDefaultGfycatAPIKey),
 		"gfycat_api_secret":                                       isDefault(*cfg.ServiceSettings.GfycatAPISecret, model.ServiceSettingsDefaultGfycatAPISecret),
+		"giphy_sdk_key":                                           *cfg.ServiceSettings.GiphySdkKey,
```

Comment:
Did we do this one in the other branches?

Reply:
I added this here because CI complained, in others they didn't so.

Reply:
Oh maybe this was added with the original implementation in the monorepo PR

Reply:
You can add  `// telemetry: none` as a comment beside the `GiphySdkKey` declaration. I don't think we want to upload the key for telemetry.